README.md

```diff
@@ -62,5 +62,5 @@ View the possible config fields [here](https://github.com/jembi/hearth/blob/mast
 * Terminology Service `$lookup` operation - ([$lookup](https://www.hl7.org/fhir/DSTU2/valueset-operations.html#lookup))
 
 # Pro dev tips:
-* To run only specific test files use `yarn test:these-files -- test/pdqm.js`. Note the `--` is important!
+* To run only specific test files use `yarn test:these-files test/pdqm.js`. Note the `--` is important!
```

Comment:
Quick comment, the note at the end of the line can now be removed.

Reply:
Removed with this [commit](https://github.com/jembi/hearth/pull/148/commits/738cf8d99e06ca823e9544e84563fab8ee554239)